ddtrace/pin.py

```diff
@@ -144,7 +145,7 @@ def override(
     def enabled(self):
         # type: () -> bool
         """Return true if this pin's tracer is enabled."""
-        return bool(self.tracer) and (self.tracer.enabled or self.tracer._apm_opt_out)
+        return bool(self.tracer) and (self.tracer.enabled or asm_config._apm_opt_out)
```

Comment:
```suggestion
       # inline to avoid circular imports
       from ddtrace.settings.asm import config as asm_config

        return bool(self.tracer) and (self.tracer.enabled or asm_config._apm_opt_out)
```

Reply:
Sorry, what exactly is this checking for if we're also allowing a disabled tracer due to ASM standalone to return True too?